This patch lets us install all the shared libs for Tcl
into /usr/lib.  However, maybe we could hide them somewhere
else; e.g. /usr/lib/tcltk/???

--- insighttoolkit-3.6.0.orig/Wrapping/CSwig/CMakeLists.txt
+++ insighttoolkit-3.6.0/Wrapping/CSwig/CMakeLists.txt
@@ -347,7 +347,7 @@
 # Configure pkgIndex.tcl for the install tree.
 
 SET(ITK_CSWIG_SCRIPT_DIR "[file dirname [info script]]")
-SET(ITK_CSWIG_PACKAGE_DIR "[file dirname [file dirname [info script]]]")
+SET(ITK_CSWIG_PACKAGE_DIR "/usr/lib")
 SET(ITK_CSWIG_DATA_ROOT "[file dirname [file dirname [info script]]]/Data")
 SET(ITK_CSWIG_TEST_ROOT "<NO_DEFAULT>")
 CONFIGURE_FILE("${ITK_SOURCE_DIR}/Wrapping/CSwig/Tcl/pkgIndex.tcl.in"
